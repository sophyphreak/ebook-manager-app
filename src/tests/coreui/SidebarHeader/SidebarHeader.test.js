import React from 'react';
import { shallow } from 'enzyme';
import SidebarHeader from '../../../coreui/SidebarHeader/SidebarHeader';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<SidebarHeader />);
});

test('should render SidebarHeader correctly', () => {
  expect(wrapper).toMatchSnapshot();
});