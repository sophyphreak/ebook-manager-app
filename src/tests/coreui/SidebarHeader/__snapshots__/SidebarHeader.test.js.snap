// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SidebarHeader correctly 1`] = `""`;
